verification/src/config.rs

```diff
@@ -64,10 +71,18 @@ impl Default for SourcifyConfiguration {
 impl Config {
     pub fn parse() -> Result<Self, config::ConfigError> {
         let args = cli::Args::parse();
+        Self::from_file(args.config_path)
+    }
+
+    pub fn test() -> Result<Self, config::ConfigError> {
+        Self::from_file(PathBuf::from_str("test_config.toml").unwrap())
+    }
```

Comment:
It's usually considered a bad practice to add a functionality required only for testing purposes. 

In our case it seems that `Config` has too broad functionality. I suggest to extract Args parsing functionality outside of the Config and leave `Config::from_file` as the only way to initialize Config structu